src/circuits/joinsplit.tcc

```diff
@@ -196,7 +195,7 @@ class joinsplit_gadget : libsnark::gadget<FieldT> {
 
             // Witness the JoinSplit inputs
             for (size_t i = 0; i < NumInputs; i++) {
-                std::vector<merkle_authentication_node> merkle_path = inputs[i].witness_merkle_path;
```

Comment:
Are you sure you want to use field and not merkle_authentication_node?

Reply:
Since it is just an alias of `Field` (see https://github.com/clearmatics/zeth/blob/mimc-integration/src/types/merkle_tree.hpp#L38) we can avoid to use it. Previously it aliased `libff::bit_vector` and it made sense since we got more compact code.